include/svnlook.php

```diff
@@ -861,6 +861,47 @@ function listFileContents($path, $rev = 0, $peg = '') {
 		}
 	}
 
+	// }}}
+	
+	// {{{ listReadmeContents
+	//
+	// Parse the readme.md file
+	function listReadmeContents($path, $rev = 0, $peg = '') {
+		global $config;
+
+		// Make this Configurable ??
+		$file = "README.md";
+
+		if($this->isFile($path.$file) != True)
+		{
+			return;
+		}
+
+		// Check for the flag for markdown Move this elsewhere?? or better to not added in composer
+		if (!defined('USE_AUTOLOADER')) {
+			require_once 'Parsedown.php';
+		}
+		else {
+			require_once 'include/parsedown.php';
+		}
+		// Capital P and small p to differentiate one used from PEAR or Composer (P) and (p) is from local include
+		$Parsedown = new Parsedown();
+
+		$cmd = $this->svnCommandString('cat', $path.$file, $rev, $peg);
+
+		if ($result = popenCommand($cmd, 'r')) 
```

Comment:
I suggest early aborts again in case of an error. How the rendered result is printed to the browser might easily become lengthier.

```PHP
$cmd = $this->svnCommandString('cat', $path.$file, $rev, $peg);
if (!($result = popenCommand($cmd, 'r')))
{
    return;
} 

echo('<div id="wrap">');
[...]
```

Reply:
Done.